<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContremForm</class>
 <widget class="QDialog" name="ContremForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>643</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Convex Hull Continuum Removal</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="9" column="1" colspan="9">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="9">
    <widget class="QToolButton" name="btnROI">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Spectral Dataset</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Mask</string>
     </property>
    </widget>
   </item>
   <item row="14" column="1" colspan="9">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnRun">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Run</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnHelp">
       <property name="text">
        <string>Help</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="11" column="1" colspan="9">
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item row="5" column="9">
    <widget class="QToolButton" name="btnOutput">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="12" column="1" colspan="9">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="9">
    <widget class="QToolButton" name="btnSpectra">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Output File</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>First Wavelength</string>
     </property>
    </widget>
   </item>
   <item row="5" column="8">
    <widget class="QComboBox" name="cboOutputType"/>
   </item>
   <item row="7" column="1">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Last Wavelength</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="7">
    <widget class="QLineEdit" name="txtROIFile"/>
   </item>
   <item row="1" column="7">
    <widget class="QSpinBox" name="spnWLLastCol">
     <property name="maximum">
      <number>9999</number>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Header Rows</string>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QSpinBox" name="spnWLFirstCol"/>
   </item>
   <item row="5" column="2" colspan="6">
    <widget class="QLineEdit" name="txtOutputFile"/>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Sample Points</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2" colspan="7">
    <widget class="QLineEdit" name="txtSamplePoints"/>
   </item>
   <item row="6" column="2">
    <widget class="QComboBox" name="cboMinWL">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>First Column</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QComboBox" name="cboMaxWL">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Last Column</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QSpinBox" name="spnWLHeaderRows"/>
   </item>
   <item row="1" column="8">
    <widget class="QCheckBox" name="chkWLTranspose">
     <property name="text">
      <string>Transpose</string>
     </property>
    </widget>
   </item>
   <item row="4" column="9">
    <widget class="QToolButton" name="btnSamplePoints">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="7">
    <widget class="QLineEdit" name="txtSpectraFile">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
